File: aula3/exercicio1.py
#!/usr/local/bioinformatic/conda3/bin/python

seqDNA = input("Digite sua sequencia de DNA\n")
seqDNA = seqDNA.upper()

seqRNA = ()

print(f"Sequencia de DNA = {seqDNA}")
print("Sequencia de RNA = ", end="")
for i in seqDNA:
    if (i =="T"):
        print("U",end="")
    else:
        print(i, end="")

print("")
print("Sequencia de DNA convertida para RNA")

File: aula4/exercicio1.R
#!/usr/local/bin/Rscript

cat("Digite sua sequencia de DNA\n")
seqDNA <- readLines(con = "stdin", n = 1)
seqDNA <- toupper(seqDNA)

cat(sprintf("Sequencia de DNA = %s\n", seqDNA))
cat("Sequencia de RNA = ")

chars <- strsplit(seqDNA, NULL)[[1]]
for (i in chars) {
  if (i == "T") {
    cat("U")
  } else {
    cat(i)
  }
}

cat("\n")
cat("Sequencia de DNA convertida para RNA\n")
